tendrl/notifier/notification/handlers/mail_handler.py

```diff
@@ -173,7 +176,20 @@ def get_mail_client(self):
     def dispatch_notification(self, alert):
         server = None
         try:
-            self.set_destinations()
+            user_ids = self.set_destinations()
+            self.get_alert_destinations(user_ids)
```

Comment:
Its better to name this function as `populate_alert_destinations` and not `get_alert_destinations` because its not returning anything